spec/requests/api/go_cardless/go_cardless_spec.rb

```diff
@@ -401,14 +401,19 @@
 
         shared_examples 'successful subscription' do
           it 'passes the correct data to the GoCardless Payment SDK' do
+
             sdk_params[:params] = sdk_params[:params].merge(
               name: "donation",
-              interval_unit: "monthly"
+              interval_unit: "monthly",
+              start_date: Date.parse("2016/02/20")
             )
             subscriptions_service = instance_double(GoCardlessPro::Services::SubscriptionsService, create: double(id: 'asdf'))
             allow_any_instance_of(GoCardlessPro::Client).to receive(:subscriptions).and_return(subscriptions_service)
             expect(subscriptions_service).to receive(:create).with(sdk_params)
-            subject
+
+            Timecop.freeze('2016/01/21') do
```

Comment:
I think these changes change the intent of this test. I think it would be nice to just duplicate the test, leaving the origin al one and having this new one explicitly say it tests that the charge date is put off to the `Settings.gbp_charge_date`